features/gebruik_in_lopende_tekst.feature

```diff
@@ -1,189 +1,651 @@
+  - gebruikInLopendeTekst wordt voorafgegaan door "mevrouw", "de heer" of de voorletters, afhankelijk van de geslachtsaanduiding:
+    | geslachtsaanduiding | gebruikInLopendeTekst begint met (GA) |
+    | M                   | de heer                               |
+    | V                   | mevrouw                               |
+    | O                   | VL                                    |
+  - gebruikInLopendeTekst wordt samengesteld uit naamcomponenten van de persoon en eventuele (ex)partner op basis van de aanduidingNaamgebruik:
+    | aanduidingNaamgebruik | gebruikinlopendetekst |
+    | E                     | GA VV GN              |
+    | P                     | GA VP GP              |
+    | V                     | GA VP GP-VV GN        |
+    | N                     | GA VV GN-VP GP        |
+  - Een voorvoegsel wordt met kleine letters wordt geschreven
+  - De eerste naamcomponent in gebruikInLopendeTekst anders dan "mevrouw" of "de heer" begint met een hoofdletter
+  - Wanneer voorletters zijn opgenomen in de gebruikinlopendetekst, het voorvoegsel van de eerste naam in de gebruikinlopendetekst met alleen kleine letters wordt geschreven
+  - Wanneer een naamcomponent geen of een lege waarde heeft, wordt de overbodige spatie niet opgenomen: niet starten met een spatie, niet eindigen met een spatie, geen dubbele spatie, geen spatie na streepje
+
+  Abstract Scenario: gebruikInLopendeTekst bij geslachtsaanduiding <voorbeeld>
+    Gegeven het systeem heeft een persoon met de volgende gegevens
+    | naam                | waarde     |
+    | burgerservicenummer | 999992934  |
+    | geslachtsaanduiding | <geslacht> |
+    En de persoon heeft de volgende naam gegevens
+    | naam                                 | waarde      |
+    | voornamen (02.10)                    | Jo Rene     |
+    | adellijke titel of predikaat (02.20) |             |
+    | voorvoegsel (02.30)                  |             |
+    | geslachtsnaam (02.40)                | Groenen     |
+    | aanduidingNaamgebruik (61.10)        | E           |
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 999992934                       |
+    | fields              | naam.gebruikinlopendetekst      |
+    Dan bevat de persoon met burgerservicenummer '999992934' de volgende 'naam' gegevens
+    | naam                  | waarde                  |
+    | gebruikinlopendetekst | <gebruikinlopendetekst> |
+
+    Voorbeelden:
+    | voorbeeld | geslacht | gebruikInLopendeTekst |
+    | man       | M        | de heer Groenen       |
+    | vrouw     | V        | mevrouw Groenen       |
+    | onbekend  | O        | J.R. Groenen          |
+
+  Scenario: gebruikInLopendeTekst van een alleenstaande persoon met aanduidingNaamgebruik "E"
```

Comment:
Waarin verschilt deze scenario met de vorige scenario? Hier wordt specifiek aanduidingNaamgebruik "E" genoemd en dat is de vorige ook. Ik zou denken dat hier een voorvoegsel scenario wordt beschreven, maar dat weet ik niet 100% zeker

Reply:
het doel van het scenario is verschillend. In veel opzichten is deze ook hetzelfde als het eerste voorbeeld in het volgende scenario. 

Het eerste scenario toont hoe gebruikInLopendeTekst eruit ziet bij verschillend geslacht van de persoon. 
Het tweede scenario toont (test) gebruikInLopendeTekst voor een alleenstaande persoon (die geen (ex)partner heeft).
Het derde scenario toont (test) gebruikInLopendeTekst voor een persoon met partner.

De twee scenario's tonen dus een andere conditie/aspect van de regel. Vind je dat niet nodig? Of kan ik dat op een andere manier duidelijker maken?

Reply:
is wat mij betreft nu goed. Het kan altijd later worden aangescherpt.